lib/membrane/rtp/session/receive_bin.ex

```diff
@@ -188,11 +190,28 @@ defmodule Membrane.RTP.Session.ReceiveBin do
 
   @impl true
   def handle_pad_removed(Pad.ref(:rtcp_output, _ref), _ctx, state) do
-    {{:ok, remove_child: :receiver_reporter}, %{state | receiver_reporter?: false}}
+    {{:ok, stop_timer: :rtcp_report_timer, remove_child: :rtcp_forwarder}, state}
   end
 
   @impl true
-  def handle_notification({:new_rtp_stream, ssrc, pt_num}, :ssrc_router, state) do
+  def handle_tick(:rtcp_report_timer, _ctx, state) do
+    {maybe_report, remote_ssrcs, report_data} =
+      Session.Report.init_report(state.ssrcs, state.rtcp_report_data)
+
+    stats_requests = Enum.map(remote_ssrcs, &{{:rtp_stream_bin, &1}, :send_stats})
+
+    report_messages =
+      case maybe_report do
+        {:report, report} -> [rtcp_forwarder: {:report, report}]
+        :no_report -> []
+      end
+
+    actions = Enum.map(report_messages ++ stats_requests, &{:forward, &1})
```

Comment:
Forward action will not accept list? If not, this should be added to the issue in core regarding a rewrite of this action

Reply:
You mean forward should support forwarding the same message to multiple children or a list of {child, message} tuples or both?

Reply:
> a list of {child, message} tuples